### ISSUE TITLE
[all payment methods] informative overlay

### ISSUE THREAD

**Issue by frosso** [woocommerce/woocommerce-gateway-stripe#1690]
## Description

![screen-shot-2021-08-13-at-2 34 56-pm](https://user-images.githubusercontent.com/273592/129619418-f6d97f67-614a-4dbc-a23c-4e2ae033aee5.png)

An overlay is displayed in the "all payment methods" section.
The overlay provides some information about the Stripe gateway.

**Comment by cesarcosta99** [comment]
@jarekmorawski is this overlay something we still would like to display in this screen?

**Comment by jarekmorawski** [comment]
Yes! Ideally, it'd be part of a larger spotlight component described in detail here: pbIJXs-RO.
